src/config.js

```diff
@@ -1,10 +1,13 @@
-var config = {
-    "title": "OpenEvent",
-    "apiBaseGetUrl": "http://springboard.championswimmer.in/api/v1/",
-    "apiTwitterGetUrl":"http://loklak.org/api/search.json?q=fossasia&callback=JSON_CALLBACK",
-    "eventId": "1",
-    "use_testApi": true,
-    "themeColorPrimary": "yellow",
-    "themeColorAccent": "teal"
+
+    "use strict";
+const config = {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["config" is defined but never used](https://www.codacy.com/app/triplez/open-event-webapp/file/2431610236/issues/source?bid=3233857&fileBranchId=3238420#l3)